index.html

```diff
@@ -901,6 +901,21 @@ <h2>Generic DID URL Parameters</h2>
 <a data-lt="ascii string">ASCII string</a>.
               </td>
             </tr>
+
+            <tr>
+              <td>
+<code>relative-ref</code>
+              </td>
+              <td>
+A relative URI reference according to <a
+data-cite="RFC3986#section-4.2">RFC3986 Section 4.2</a>
+that identifies a resource at a <a>service endpoint</a>, which is selected from
```

Comment:
Section 3.2.3 about DID Path says:

> A DID path SHOULD be used to address resources available through a service endpoint.

How someone could decide whether he must use DID Path or _relative-ref_ parameter when he creates a relative url into a service endpoint? This has to be clarified here and in Section 3.2.3

Reply:
Good point. Based on previous discussions on the topic, the sentence in 3.2.3 should be removed, since the `relative-ref` parameter will be used for this. There have even been proposals to remove DID Path altogether, but then the consensus was that DID Path may have other uses that have nothing to do with service endpoints. I will update this PR accordingly.